posts/atomic-design-bootstrap.md

```diff
@@ -0,0 +1,154 @@
+_Bootstrap grid system example from the website_
+
+One of these elements is the [grid system](https://getbootstrap.com/docs/4.4/layout/grid), because it allows a grid simulation with 12 columns and to manage some responsive design issues. Even if gutters (space between columns: 30px by default) may have a lot of effects on the building of a grid, they can be removed using the `no-gutters` modifier.
+
+Related to `Components`, the grid has a less impact on a perceptive and functional value. It is used to structure elements inside columns and can be used on the `organism` part to manage `molecules` and `atoms`.
+
+## Utilities
+
+Some elements are used as utilities, they usually have a lower graphical value than `Components` and are used by composition..
+
+As an example, we will use [text](https://getbootstrap.com/docs/4.4/utilities/text).
+
+This utility lets you do generics actions like:
+
+* alignment
+* prevent line return and truncate
+* transform to lowercase, uppercase …
+* change weight and use italic
+* change font style to monospace
+
+Some rules are more generic than others: where alignment is common and will have a small effect on style, the specific font choice for something like monospace may have a greater impact:
+
+```css
+.text-monospace {
+    font-family: SFMono-Regular,Menlo,Monaco,Consolas,"Liberation Mono","Courier New",monospace;
+}
+```
+
+May be you can think it is not an important effect, but, depending on the utilities, this type of effect may vary, this is the point of [colors](https://getbootstrap.com/docs/4.4/utilities/colors) which has its own semantic and refers to _Bootstrap_ branding.
+
+You must be pragmatic to choose utilities.
+
+# Go further with Sass
+
+Extending and customising _Bootstrap_ can be done using  [Sass (SCSS)](https://github.com/twbs/bootstrap) [sources](https://sass-lang.com). It is possible to change container size depending on window size, colors, and there are many other exposed variables using `!default` Sass principle. Moreover, file fragmentation lets you use parts of _Bootstrap_ like using only the grid system for example by importing the Sass file you want to use.
+
+In this case, you can use them for:
+
+* `Components` : Changing colors for perception
+* `grid system` : Changing columns number to fit what you need
+
+More information on what is available to extend _Bootstrap_ are on the [Theming Bootstrap](https://getbootstrap.com/docs/4.4/getting-started/theming) page.
+
+I recommend you to not override too many properties of _Bootstrap_ because your coupling can impact your maintainability and version upgrades.
+
+# Conclusion
+
+There are a lot of ways to use _Bootstrap_ in an _Atomic Design_ project. With the experience and depending on the context, we learn how not to override, not to use built-in __Bootstrap__ components without thinking about how they will be used and how they are perceived. Sometimes it is not needed to add to the framework because it may not add value.
```

Comment:
```suggestion
There are a lot of ways to use _Bootstrap_ in an _Atomic Design_ project. With the experience and depending on the context, we learn how not to override and not to use built-in __Bootstrap__ components without thinking about how they will be used and how they are perceived. Sometimes it is not necessary to add to the framework because it may not add value.
```